docs/user/accumulo_datastore.rst

```diff
@@ -88,6 +88,13 @@ at data store, feature, or individual attribute levels.
 
 See :ref:`accumulo_visibilities` for more information.
 
+Scan Authorizations
+-------------------
+
+The GeoMesa Accumulo data store provides a Java SPI for custom handling of user authorizations per query.
+For simple use cases, authorizations can be set at the data store level through a configuration parameter.
```

Comment:
Ah, you're talking about the "auths" data store parameter. Sorry, I was overthinking it. I think it might be a good idea to take the text here:

https://github.com/geomesa/geomesa-tutorials/tree/master/geomesa-examples-authorizations#authorizations-in-geomesa

and move/copy it here. Alternately we should put a more precise link to this section of this tutorial rather than linking to the "security" section in the tutorials table of contents.